Author: "Zak B. Elep" <zakame@zakame.net>
Last-Updated: Sat, 31 Mar 2012 01:01:48 +0800
Description: Fix doc install path
  Use /usr/share/doc/dbacl as documentation install base.
diff --git a/doc/Makefile.am b/doc/Makefile.am
index 0ce4cd2..cd23153 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -1,5 +1,5 @@
 SUBDIRS = chess
-documentationdir = $(pkgdatadir)/doc
+documentationdir = /usr/share/doc/dbacl
 documentation_DATA = dbacl.ps costs.ps tutorial.html email.html is_it_working.html \
 	sample1.txt sample2.txt sample3.txt sample4.txt sample5.txt sample6.txt \
 	toy.risk example1.risk example2.risk example3.risk \
@@ -7,4 +7,4 @@ documentation_DATA = dbacl.ps costs.ps tutorial.html email.html is_it_working.ht
 EXTRA_DIST = dbacl.ps costs.ps tutorial.html email.html is_it_working.html \
 	sample1.txt sample2.txt sample3.txt sample4.txt sample5.txt sample6.txt \
 	toy.risk example1.risk example2.risk example3.risk \
-	prop.pl japanese.txt
\ No newline at end of file
+	prop.pl japanese.txt
diff --git a/doc/chess/Makefile.am b/doc/chess/Makefile.am
index d5bdff7..d39a61e 100644
--- a/doc/chess/Makefile.am
+++ b/doc/chess/Makefile.am
@@ -1,4 +1,4 @@
-documentationdir = $(pkgdatadir)/doc/chess
+documentationdir = /usr/share/doc/dbacl/chess
 documentation_DATA = spam_chess.html dce.sh \
 	csfpc1.png csfpc2.png csfpc3.png down.png spoiler.png \
 	dce-basic.sh dce-1.sh dce-2.sh dce-3.sh \
